.emacs.d/lisp/system/darwin.el

```diff
@@ -38,7 +47,7 @@ stuff.  This also respects the user's shell decision."
 
 (if (eq system-type 'darwin)
     (progn
-      (kotct/darwin-swap-command-option)
+      (kotct/darwin-maybe-swap-command-option)
```

Comment:
Unless I'm mistaken, this will test the value of `kotct/darwin-swap-command-option` when it's run, so it'll always be `t`. If a user sets it in their personal config, that won't have any effect. I propose running `kotct/darwin-maybe-swap-command-option` on `after-init-hook` to make sure any changes to the variable in personal configs are accounted for.